test/validators.js

```diff
@@ -5174,12 +5174,14 @@ describe('Validators', () => {
       {
         locale: 'pt-BR',
         valid: [
-          '+55-12-996551215',
```

Comment:
Could you please explain this regression?

Reply:
Brazilian number formats never use dashes between numbers.